WORKSPACE

```diff
@@ -82,3 +82,10 @@ http_file(
     # .bazelversion uses 4.0
     urls = ["https://github.com/bazelbuild/buildtools/releases/download/3.5.0/buildifier.mac"],
 )
+
+load(
```

Comment:
nit: Should we add a comment here?

Reply:
I think the name is pretty self documenting, but not opposed to a comment  `load_external_test_dependency`